docker-ubuntu.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+#Exit on error
+set -e
+
+sudo -v
+
+#Add the GPG key for the official Docker repository to the system
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+
+#Add the Docker repository to APT sources
+sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+
+#Update the package database with Docker packages from the newly added repo
+sudo apt update -y
+
+#To install from the Docker repo instead of the default Ubuntu16.04 repo
+apt-cache policy docker-ce
+
+#Install Docker
+sudo apt-get install -y docker-ce
+
```

Comment:
Add docker installation verification.

Reply:
Yea, I have added that verification at the end of the script. If you see its output as the latest version of Docker, it means the whole execution was successful. Do you mean to let user know the installation is successful by displaying some user-friendly message?

Reply:
Yup exactly like that. Because when we will merge everything inside setup.sh he may not see that. So a good message may do the work.